src/rpc/core_rpc_server.cpp

```diff
@@ -2718,8 +2718,33 @@ namespace cryptonote
 
             entry.quorum.validators.reserve(quorum->validators.size());
             entry.quorum.workers.reserve(quorum->workers.size());
-            for (crypto::public_key const &key : quorum->validators) entry.quorum.validators.push_back(epee::string_tools::pod_to_hex(key));
-            for (crypto::public_key const &key : quorum->workers)    entry.quorum.workers.push_back(epee::string_tools::pod_to_hex(key));
+
+            auto const &service_node_list = m_core.get_service_node_list();
+            uint64_t const now = time(nullptr);
+
+            service_node_list.for_each_service_node_info_and_proof(
+             quorum->validators.begin(),
+             quorum->validators.end(),
+             [&](auto& public_key, auto&, auto& proof) {
+               cryptonote::COMMAND_RPC_GET_QUORUM_STATE::quorum_validator validator;
+
+               validator.hash = epee::string_tools::pod_to_hex(public_key);
```

Comment:
This is better named as `pkey` or `pub_key`. We're also moving away from epee related code, the equivalent here would be `lokimq::to_hex(tools::view_guts(public_key));`

There's a helper function somewhere in the codebase that combines that step into 1 on the dev branch iirc. Could this commit be rebased to dev? As this PR uses the old RPC paradigm that has been upgraded in dev and would require fixing up again for the next HF.

Reply:
Yeah I will rebase dev, use the new methods, and update the public_key reference